src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/api/model/sysconsync/Prisoner.kt

```diff
@@ -0,0 +1,85 @@
+package uk.gov.justice.digital.hmpps.personrecord.api.model.sysconsync
+
+import java.time.LocalDate
+
+data class Prisoner(
+  val title: String? = null,
+  val firstName: String? = null,
+  val middleName1: String? = null,
+  val middleName2: String? = null,
+  val lastName: String? = null,
+  val dateOfBirth: LocalDate? = null,
+  val birthplace: String? = null,
+  val nationality: String? = null,
+  val multipleNationalities: List<String> = emptyList(),
+  val religions: List<Religion> = emptyList(),
+  val crn: String? = null,
+  var prisonNumber: String? = null,
+  val ethnicity: String? = null,
+  val race: String? = null,
+  val sexualOrientation: String? = null,
+  val sex: String? = null,
+  val interestToImmigration: Boolean? = null,
+  val disability: Boolean? = null,
+  val currentlyManaged: Boolean? = null,
+  val sentenceStartDate: LocalDate? = null,
```

Comment:
Do we have the ability to get multiple historic sentence dates, or, are we limited to 1 from Nomis ?

Reply:
Good point, one to raise in the meeting tomorrow.